addons/web/static/src/views/relational_model.js

```diff
@@ -3556,6 +3556,9 @@ export class RelationalModel extends Model {
     }
 
     hasData() {
+        if (this.root.groups) {
+            return this.root.groups.some((group) => group.count > 0 || group.quickCreateRecord);
```

Comment:
does it work if grouped by multiple fields?

Reply:
Yes, it does

![image](https://github.com/odoo/odoo/assets/47742104/70e69d3e-615d-489f-afed-2fea5e53a468)